### DISCUSSION TITLE
Replacing signal reports with country on main dashboard

### DISCUSSION THREAD

**Discussion by ofadam** [magicbug/Cloudlog#855] [Ideas / Feature Requests]
On the main dashboard that loads upon login, I find myself often clicking on individual QSOs to remember the country. Instead of signal reports on that page with the limited horizontal space, could it be the country?

I can start on a PR if there's support for the idea.

**Comment by AndreasK79** [comment]
@ofadam You get my vote for a change.

**Reply by ghost** [reply]
I would also prefer that as the default.

**Comment by ghost** [comment]
We could merge the sent/received values into one column to avoid losing any existing information.
